llvm/include/llvm/SYCLLowerIR/EmitSYCLNativeCPUHeader.h

```diff
@@ -0,0 +1,37 @@
+//===------- EmitSYCLHCHeader.h - Emits the SYCL Native CPU helper header
+//-------===//
```

Comment:
the line was too long so it got wrapped, please reduce the length

Reply:
Reduced line length, thank you